hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestHoodieSparkSqlWriter.scala

```diff
@@ -1250,6 +1251,7 @@ class TestHoodieSparkSqlWriter {
 
     assertTrue(beforeWriteInstant.toEpochMilli < writeInstant.toEpochMilli,
       s"writeInstant(${writeInstant.toEpochMilli}) must always be greater than beforeWriteInstant(${beforeWriteInstant.toEpochMilli}) if writeInstant was generated with UTC timezone")
+    TimeZone.setDefault(defaultTimezone)
```

Comment:
Should wrap with `try ... finally` incase some errors throw out before this code?

Reply:
Makes sense.  Fixed.